selfdrive/ui/qt/maps/map_settings.h

```diff
@@ -69,31 +66,27 @@ class MapSettings : public QFrame {
 class DestinationWidget : public QPushButton {
   Q_OBJECT
 public:
-  explicit DestinationWidget(QWidget *parent = nullptr);
-  void set(const QJsonObject &location, bool current = false);
-  void unset(const QString &label, bool current = false);
+  enum Type { Home, Work, Favorite, Recent, Current };
```

Comment:
is there any reason to make `Type` public when the `typeIcon` helper is private?

Reply:
`class MapSettings` need access `Type` to create different `DestinationWidget`:

current_widget = new DestinationWidget(DestinationWidget::Current, this)
home_widget = new DestinationWidget(DestinationWidget::Home, this)
work_widget = new DestinationWidget(DestinationWidget::Work, this)

Reply:
oh!